src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -170,6 +170,7 @@
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/button.state.toggle.js'
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.version.edit.conflict.js'
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.location.bookmark.js'
+        '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.main_translation.update.js'
```

Comment:
In JS files we do not use the convention with `_`. Should be `admin.main.translation.update.js`.